tomb-common/Cargo.toml

```diff
@@ -30,11 +30,12 @@ sha1 = "^0.10"
 sha2 = "^0.10"
 test-case = "^3"
 thiserror = "^1"
-uuid = { version = "^1", features = ["serde"] }
+uuid = { workspace = true }
 wasm-bindgen = "^0.2"
 unsigned-varint = "^0.7"
 bytes = "1.5.0"
 async-recursion = "1.0.5"
+base64-url = "2.0.0"
```

Comment:
When adding new dependencies, pin it to the stable semver version (for version > 1.0.0 this is the major version, for version < 1.0.0 it is the minor version. Pinning 2.3.1 would be "^2", pinning 0.34.1 would be "^0.34". This allows flexibility when updating packages and between different dependencies that are pinned to slightly different versions.